helloworld/test/index-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const chai = require('chai');
+const greet = require('../index.js');
```

Comment:
Work on your naming conventions as this gets really confusing. Since you already have a ```greet``` module that ties to a module specifically named ```greet.js``` it took awhile to realize this test's ```greet``` was NOT linked to the ```greet.js``` module, but to the ```index.js``` module. 

Since using the word ```index``` can get confusing too, I'd recommend just naming this ```main``` in the future if you're ever exporting an ```index.js``` module.